Classes/Utility/AttributeUtility.php

```diff
@@ -41,6 +41,58 @@ public static function findAttribute(int $attributeId, string $selectFields = '*
         );
     }
 
+    /**
+     * Returns all attribute id's for corresponding product.
```

Comment:
It doesn't just return the IDs. The default $selectFields is "*", so it actually returns all fields.

```suggestion
     * Returns attributes for corresponding product.
```